### ISSUE TITLE
JavaScript text input panel is not displayed

### ISSUE THREAD

**Issue by danschlet** [SafeExamBrowser/seb-mac#181]
### Discussed in https://github.com/SafeExamBrowser/seb-mac/discussions/163

<div type='discussions-op-text'>

<sup>Originally posted by **juliussaputra** December  1, 2021</sup>
am currently testing the SEB for our basic programming exams.
For the exam we are using a third party IDE ([pythonandturtle.com](https://pythonandturtle.com/)).

Problem: The input prompt window from the IDE is not shown in SEB MAC(**3.0**) but in windows (SEB **3.3.1**) it is working without any problem.
Both case were tested with the same SEB config file on MAC OS Monterey 12.0.1 .

Screenshots from both cases are attached:
[SEB_Popup_Windows](https://user-images.githubusercontent.com/17081818/144246947-74d86130-e8cb-4496-b9d6-8c46c0bc67a9.jpeg)
[SEB_Popup_Mac](https://user-images.githubusercontent.com/17081818/144246959-15f9e529-3808-4839-95db-70c66a0a93ec.jpeg)

For the test, I have tried SEB with the w3schools link as well: https://www.w3schools.com/js/tryit.asp?filename=tryjs_prompt. The result was the same, the prompt window is just not shown and return empty string value as a result.

is there any specific setting for SEB in MAC that allows the input window? or is there any workaround for this case?

Many thanks in advance for your work!
Julius</div>

**Comment by danschlet** [comment]
The JS text input panel is implemented in SEB 3.1, the [macOS preview version 7 (3.1pre7)](https://github.com/SafeExamBrowser/seb-mac/releases) contains it. Maybe you can confirm it works for you as well.

**Comment by PhilippImhof** [comment]
As mentioned in #176, I have a similiar problem. The `prompt()` is (silently) not shown in Classic WebView mode. When using Modern mode, it even crashes SEB.

``` 
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               Safe Exam Browser [46642]
Path:                  /Applications/Safe Exam Browser.app/Contents/MacOS/Safe Exam Browser
Identifier:            org.safeexambrowser.SafeExamBrowser
Version:               3.0 (343F)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2022-03-28 07:59:41.7812 +0200
OS Version:            macOS 12.2.1 (21D62)
Report Version:        12
Anonymous UUID:        98A63FEA-9CAE-5BEA-EF29-DF5101B51A18

Sleep/Wake UUID:       B53FFDA7-D0CE-4CA7-B481-1A43E9239A9D

Time Awake Since Boot: 320000 seconds
Time Since Wake:       57 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Application Specific Information:
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Completion handler passed to -[Safe_Exam_Browser.SEBOSXWKWebViewController webView:runJavaScriptTextInputPanelWithPrompt:defaultText:initiatedByFrame:completionHandler:] was not called'
abort() called
terminating with uncaught exception of type NSException
```

3.1pre10 fixes the issue for me.
